config.js

```diff
@@ -438,6 +432,35 @@ config.DEFAULT_ACCOUNT_PREFERENCES = {
 ///////////////////////////////////
 config.REMOTE_NOOAA_NAMESPACE = `remote-${config.KUBE_APP_LABEL}`;
 
+///////////////////////////////
+// FILES RELATED             //
+///////////////////////////////
+config.INLINE_MAX_SIZE = 4096;
+
+// Namespace bucket cache expiration time
+config.NAMESPACE_CACHE_EXPIRY_MS = 60000;
```

Comment:
this name got a bit out of context overtime, 
I suggest to rename to `OBJECT_SDK_BUCKET_CACHE_EXPIRY_MS`

Reply:
ok